pydantic_settings/main.py

```diff
@@ -134,98 +153,166 @@ def parse_env_var(cls, field_name: str, raw_val: str) -> Any:
     __config__: ClassVar[Type[Config]]
 
 
-class InitSettingsSource:
+class PydanticBaseSource(ABC):
```

Comment:
please can you add a docstring here.

Reply:
should these all move to a separate `sources.py` module?

Reply:
move to `sources.py` and docstrings added